src/libs/ffmpeg/lv_ffmpeg.c

```diff
@@ -291,7 +295,7 @@ static lv_result_t decoder_open(lv_image_decoder_t * decoder, lv_image_decoder_d
     if(dsc->src_type == LV_IMAGE_SRC_FILE) {
         const char * path = dsc->src;
 
-        struct ffmpeg_context_s * ffmpeg_ctx = ffmpeg_open_file(path);
+        struct ffmpeg_context_s * ffmpeg_ctx = ffmpeg_open_file(path, &dsc->file);
```

Comment:
`ffmpeg_open_file(path, true);` ?

Reply:
fixed.
What surprised me was that no compiler detected this error.